File: icsopenapi/easyopen-sdk/sdk-csharp/EasyopenSDKCSharp/Request/CommonRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EasyopenSDKCSharp.Request
{
    public class CommonRequest : BaseRequest<EasyopenSDKCSharp.Response.CommonResponse>
    {
        public CommonRequest(string name)
        {
            this.Name = name;
        }

        public CommonRequest(string name, string version)
        {
            this.Name = name;
            this.Version = version;
        }

        public override string GetName()
        {
            return string.Empty;
        }
    }
}

File: icsopenapi/easyopen-sdk/sdk-java/src/main/java/com/easyopen/sdk/request/CommonRequest.java
package com.easyopen.sdk.request;

import com.easyopen.sdk.response.CommonResponse;

/**
 * @author tanghc
 */
public class CommonRequest extends BaseRequest<CommonResponse> {

    public CommonRequest(String name) {
        this.setName(name);
    }

    public CommonRequest(String name, String version) {
        this.setName(name);
        this.setVersion(version);
    }

    @Override
    public String name() {
        return "";
    }
}
